### ISSUE TITLE
Question about signing up[Q]

### ISSUE THREAD

**Issue by iamstarlee** [wandb/wandb#6114]
Hi, I'm trying to sign up from the "https://wandb.ai/signup", but I encountered this bug "Error creating account: entity already exists for the user 13253620092", which puzzles me a lot, could you please tell me how to solve it?

**Comment by iamstarlee** [comment]
I can't login in, it keeps saying '"Error creating account: entity already exists for the user 13253620092". Does anyone know this problem?

**Comment by iamstarlee** [comment]
I changed my email account, and it works, but still confused about this problem :-(

**Comment by ArtsiomWB** [comment]
Hi @iamstarlee! Thank you for writing in. I think what might have happened was a signup bug that our team is currently working on resolving. Would you still like to have an account with your original username, or is the new one you created okay for you?

**Comment by ArtsiomWB** [comment]
We wanted to follow up with you regarding your support request as we have not heard back from you. Please let us know if we can be of further assistance or if your issue has been resolved.

**Comment by ArtsiomWB** [comment]
Hi @iamstarlee, since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!

**Comment by sydholl** [comment]
Sydney Hollingsworth commented: 
This ticket has gone stale, please follow up or close.

**Comment by sydholl** [comment]
Artsiom Skarakhod commented: 
Hi Internal,

We wanted to follow up with you regarding your support request as we have not heard back from you. Please let us know if we can be of further assistance or if your issue has been resolved.

Best,
Weights & Biases

**Comment by sydholl** [comment]
Sydney Hollingsworth commented: 
Cannot submit ticket because type is Question.

**Comment by AkimParis** [comment]
Dear @cvphelps 

I have a similar problem. I saw that you had this bug years ago. 

I have successfully signed up for my W&B account and enjoyed a training job that worked perfectly.

When I try to connect to my account. I got the https://wandb.auth0.com then "Last time you logged in with my email" with the correct email. The website is redirecting me to "Submit this form" which closes immediately. 
Then the website automatically signs in under my account, while redirecting me to the "Sign Up" page... Very confusing! : )

Under my account, I have only 2 options:

Create your account -> Full Name, Company or Institution, and Username
Or 
Click on the Profil icon -> Log in with another account, Contact Us, and Log out

Thank you for your help!
Akim
